media/css/custom.scss

```diff
@@ -0,0 +1,496 @@
+    padding-right: 20px;
+  }
+}
+
+.container {
+  .credit {
+    margin: 20px 0;
+  }
+}
+
+#feed-range {
+  cursor: pointer;
+  padding: 5px 10px;
+  border: 1px solid $white;
+  &:hover {
+    background-color: $widget-link-highlight;
+  }
+}
+
+#rssdata ul.rss-items {
+  display: none;
+  margin: 0;
+  padding: 0;
+  li {
+    display: block;
+    margin: 0;
+    padding: 0;
+  }
+  a {
+    margin: 0;
+    display: block;
+    font: 15px / 16px $font-stack;
+    padding: 5px;
+    background: $sidebar-background;
+    color: $white;
+    text-decoration: none;
+    border-bottom: 1px solid $sidebar-border-color;
+    &:hover {
+      background: $sidebar-link-highlight;
+      color: $white;
+      text-decoration: none;
+    }
+  }
+}
+
+#rsstitle {
+  padding: 6px;
+  color: $white;
+  font: 17px / 18px bold $font-stack;
+  text-decoration: none;
+  border-bottom: 2px solid $sidebar-border-color;
+  height: 25px;
+}
+
+#rssbottom {
+  padding: 2px 6px;
+  background: $sidebar-background;
+  color: $white;
+  text-decoration: none;
+  border-top: 2px solid $sidebar-border-color;
+  height: 20px;
+}
+
+#twitter ul.twits {
+  display: none;
+  margin: 0;
+  padding: 0;
+  li {
+    display: block;
+    margin: 0;
+    padding: 0;
+  }
+  a.tw {
+    margin: 0;
+    display: block;
+    font: 15px / 16px $font-stack;
+    padding: 5px;
+    background: $sidebar-background;
+    color: $white;
+    text-decoration: none;
+    border-bottom: 1px solid $sidebar-border-color;
+    &:hover {
+      background: $sidebar-link-highlight;
+      color: $sidebar-link-highlight;
+      text-decoration: none;
+    }
+  }
+}
+
+#twitter_title {
+  padding: 6px;
+  color: $white;
+  font: 17px / 18px bold $font-stack;
+  text-decoration: none;
+  border-bottom: 2px solid $sidebar-border-color;
+}
+
+#twitter_bottom {
+  padding: 2px 6px;
+  background: $sidebar-background;
+  color: $white;
+  text-decoration: none;
+}
+
+.white {
+  color: $white;
+}
+
+a.white {
+  color: $white;
+  &:hover {
+    color: $white;
+  }
+}
+
+.article {
+  .thumbnail {
+    margin: 15px;
+  }
+}
+
+a {
+  color: $sidebar-background;
+  &:hover {
+    color: $sidebar-link-highlight;
+  }
+  &:focus {
+    outline: thin dotted $sidebar-link-highlight;
+    outline-offset: -2px;
+  }
+}
+
+.well {
+  background-color: $sidebar-background;
+  li a {
+    color: $white;
+  }
+}
+
+a.thumbnail {
+  background-color: $white;
+  &:hover {
+    background-color: $sidebar-link-highlight;
+  }
+}
+
+#sidebar {
+  .well,
+  h2,
+  a {
+    color: $white;
+  }
+  a:hover {
+    color: $sidebar-link-highlight;
+  }
+}
+
+#panel1,
+#panel2,
+#panel3 {
+  display: none;
+  padding: 5px;
+  width: 225px;
+  height: 40px;
+  background: $sidebar-background;
+  color: $white;
+}
+
+#photo-feed {
+  > .span3 {
+    height: 350px;
+    margin: 5px 5px 25px;
+  }
+  .thumbnail {
+    height: inherit;
+    & > img {
+      width: 210px;
+      height: 200px;
+    }
+    div.caption {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3